python-crud-google-spreadsheet/src/create.py

```diff
@@ -0,0 +1,18 @@
+import gspread
```

Comment:
[PEP8: Import order](https://www.python.org/dev/peps/pep-0008/#imports): `os` should come first